stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamShapes.kt

```diff
@@ -44,6 +45,7 @@ public data class StreamShapes(
     public val otherMessageBubble: Shape,
     public val inputField: Shape,
     public val attachment: Shape,
+    public val quotedAttachment: Shape,
```

Comment:
Should be at the end as it's a new parameter that we introduced. Otherwise it'll break the order for our customers